release/preview/alpine39/docker/Dockerfile

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Docker image file that describes an Alpine3.8 image with PowerShell installed from .tar.gz file(s)
+
+# Define arg(s) needed for the From statement
+ARG fromTag=3.9
+ARG imageRepo=alpine
+
+FROM ${imageRepo}:${fromTag} AS installer-env
+
+# Define Args for the needed to add the package
+ARG PS_VERSION=6.2.0-preview.3
+ARG PS_PACKAGE=powershell-${PS_VERSION}-linux-alpine-x64.tar.gz
+ARG PS_PACKAGE_URL=https://github.com/PowerShell/PowerShell/releases/download/v${PS_VERSION}/${PS_PACKAGE}
+ARG PS_INSTALL_VERSION=7-preview
+
+# Download the Linux tar.gz and save it
+ADD ${PS_PACKAGE_URL} /tmp/linux.tar.gz
+
+# define the folder we will be installing PowerShell to
+ENV PS_INSTALL_FOLDER=/opt/microsoft/powershell/$PS_INSTALL_VERSION
+
+# Create the install folder
+RUN mkdir -p ${PS_INSTALL_FOLDER}
+
+# Unzip the Linux tar.gz
+RUN tar zxf /tmp/linux.tar.gz -C ${PS_INSTALL_FOLDER}
```

Comment:
Maybe add `v` so we have a list of files that were uncompressed.

Reply:
filed #245